source/guide/3.2/plugin-host-pool.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: bt_wiki
+title: Host-Pool Plugin
+category: Plugins
+publish: true
+abstract: "Cloudify Host-Pool plugin description and configuration"
+pageord: 900
+---
+{%summary%}
+{%endsummary%}
+
+# Description
+
+The [Host Pool plugin](https://github.com/cloudify-cosmo/cloudify-host-pool-plugin) allows for the use of multiple existing hosts to be allocated for a deployment.
+It interacts with Cloudify's [Host-Pool Service](https://github.com/cloudify-cosmo/cloudify-host-pool-service), which performs the book-keeping as to which hosts are allocated, free, or un-reachable.
+
+# Host-Pool Service
+
+The Host-Pool Service is a web service designed for managing a large pool of hosts to be used by cloudify deployments.
+The Host-Pool-Plugin will make calls to this service each time a new host
+needs to be provisioned/terminated.
+
+To make the installation of this service easy, we have made it available as a regular cloudify node type.
+
+## cloudify.nodes.HostPoolService
+
+**Derived From:** [cloudify.nodes.SoftwareComponent](reference-types.html)
+
+**Properties:**
+
+  * `pool` relative path to a pool configuration file. This is where you define all the hosts participating in the pool.
+  * `directory` the directory to run the service from inside the host. Defaults to `/tmp/cloudify-host-pool-service`
+  * `port` the port to run the service on. Defaults to `8080`
+  * `source` the source code of the service. Defaults to `https://github.com/cloudify-cosmo/cloudify-host-pool-service/archive/master.zip`
+
+**Mapped Operations:**
+
+  * `cloudify.interfaces.lifecycle.create` creates the necessary directories and installs dependencies.
+  * `cloudify.interfaces.lifecycle.configure` creates the service configuration file based on information in the properties.
+  * `cloudify.interfaces.lifecycle.start` starts the service.
+  * `cloudify.interfaces.lifecycle.stop` stops the service.
+  * `cloudify.interfaces.lifecycle.delete` deletes the service working directory.
+
+**Attributes:**
+
+  * `private_endpoint` the url of the service. this URL is only accessible from within the same network as the host itself.
```

Comment:
private_endpoint --> endpoint and explain that it uses the ip runtime property the plugin sets